main/api/src/config.ts

```diff
@@ -25,6 +25,9 @@ const ConfigSchema = z.object({
     region: z.string().optional(),
     endpoint: z.string().optional(),
   }),
+  table: z.object({
+    name: z.string()
+  })
```

Comment:
That config can go into dynamo configuration and could be. `tableName: z.string()`